pkg/backend/backend.go

```diff
@@ -57,6 +63,21 @@ func NewBackend(ctx context.Context, log *logrus.Entry, env env.Interface, db *d
 	b.ocb = &openShiftClusterBackend{backend: b}
 	b.sb = &subscriptionBackend{backend: b}
 
+	if len(os.Getenv("MAX_WORKERS")) > 0 {
```

Comment:
set maxWorkers, maxDequeueCount to their defaults above the os.Getenv's, then you don't need the pointer